src/loaders/bugs.js

```diff
@@ -48,9 +49,21 @@ export default () => {
       })
     )
   );
+  const comments = new DataLoader(ids =>
```

Comment:
I only had you rename this to `ids` because that was the only value you were passing, but now you're passing `query` as well, so rename this back to `queries`. :smile: